website/src/components/CustomerTestimonials/index.tsx

```diff
@@ -102,59 +114,76 @@ const TestimonialBox = ({
             >
               Customers{" "}
-              <FaHeart className="text-red-500 w-12 h-12 mx-1 inline-block" />{" "}
+              <FaHeart className="text-primary-450 w-12 h-12 mx-1 inline-block" />{" "}
               us,
             </h3>
-            <p className="text-neutral-500 text-2xl md:text-3xl font-medium">
+            <h3 className="text-neutral-600 text-3xl md:text-4xl leading-12 xl:text-5xl text-wrap sm:text-nowrap tracking-tight font-medium mb-6">
               and we love them back.
-            </p>
+            </h3>
+            <ActionLink href="/contact/sales" color="white">
+              Book a demo
+            </ActionLink>
           </div>
         </div>
-
-        <div
-          ref={scrollRef}
-          className="sm:fade-side flex gap-12 px-8 sm:px-0 w-full snap-x snap-mandatory pb-24 sm:pb-12 overflow-x-auto dark-scroll"
-        >
-          {customerData.map((item, index) => (
-            <TestimonialBox key={index} {...item} />
-          ))}
-        </div>
-        <div className="absolute bottom-12 left-1/2 -translate-x-1/2 sm:left-auto sm:translate-x-0 sm:bottom-auto sm:right-16 sm:top-8 flex items-center gap-16">
-          <button onClick={scrollLeft}>
-            <HiArrowLeft className="hover:bg-[#1b1b1d] cursor-pointer rounded-full border p-1.5 text-neutral-50 w-8 h-8" />
-          </button>
-          <button onClick={scrollRight}>
-            <HiArrowRight className="hover:bg-[#1b1b1d] cursor-pointer rounded-full border p-1.5 text-neutral-50 w-8 h-8" />
-          </button>
+        <div className="flex flex-col items-center px-4 sm:px-16 lg:px-0 gap-4 lg:gap-0 lg:flex-row">
+          <TestimonialBox
+            authorTitle={customerData[1].authorTitle}
+            desc={customerData[1].desc}
+            companyName={customerData[1].companyName}
+            fontSize="lg"
+            href={customerData[1].href}
+            authorImage={customerData[1].authorImage}
+            authorName={customerData[1].authorName}
+          />
+          <div className="flex flex-col pl-0 lg:pl-2 lg:gap-2 gap-4">
+            <TestimonialBox
+              fontSize="md"
+              href={customerData[0].href}
+              authorTitle={customerData[0].authorTitle}
+              desc={customerData[0].desc}
+              companyName={customerData[0].companyName}
+              authorImage={customerData[0].authorImage}
+              authorName={customerData[0].authorName}
+            />
+            <TestimonialBox
+              fontSize="md"
+              href={customerData[2].href}
+              authorTitle={customerData[2].authorTitle}
+              desc={customerData[2].desc}
+              companyName={customerData[2].companyName}
+              authorImage={customerData[2].authorImage}
+              authorName={customerData[2].authorName}
+            />
+          </div>
```

Comment:
(nit): It would be slightly more intuitive to rearrange the data so that this is in order from `0, 1, 2`

Reply:
i think i chose this way bc specifically mark had the shortest testimonial and would be the one that was most likely to be read

Reply:
but yea i guess now since there is no map function we can just directly write the names

Reply:
👍